test/search_index_test.dart

```diff
@@ -76,7 +77,7 @@ class C {}
     var classItem = jsonIndex.named('index_json.C');
 
     expect(classItem['kind'], equals(Kind.class_.index));
-    expect(classItem['overriddenDepth'], equals(0));
+    expect(classItem['overriddenDepth'], null);
```

Comment:
IIUC we generally prefer `isNull` to the `null` literal in the analyzer code base(s). I think?

Reply:
SGTM!